src/OrchardCore.Modules/OrchardCore.Workflows/Http/Scripting/HttpMethodsProvider.cs

```diff
@@ -62,11 +67,56 @@ public HttpMethodsProvider(IHttpContextAccessor httpContextAccessor)
                     return body;
                 })
             };
+
+            _requestFormMethod = new GlobalMethod
+            {
+                Name = "requestForm",
+                Method = serviceProvider => (Func<string, object>) (field =>
+                {
+                    object result;
+                    if (httpContextAccessor.HttpContext.Request.Form.TryGetValue(field, out var values))
+                    {
+                        if (values.Count == 0)
+                        {
+                            result = null;
+                        }
+                        else if (values.Count == 1)
+                        {
+                            result = values[0];
+                        }
+                        else
+                        {
+                            result = values.ToArray();
+                        }
+                    }
+                    else
+                    {
+                        result = null;
+                    }
+                    return result;
+                })
+            };
+
+            _queryStringJSONMethod = new GlobalMethod
+            {
+                Name = "queryStringJSON",
+                Method = serviceProvider => (Func<JObject>)(() =>
+                    new JObject((from param in httpContextAccessor.HttpContext.Request.Query
+                                 select new JProperty(param.Key, JArray.FromObject(param.Value.ToArray()))).ToArray()))
+            };
+
+            _requestFormJSONMethod = new GlobalMethod
+            {
+                Name = "requestFormJSON",
```

Comment:
JSON - because query param or form field names may not be valid Javascript identifiers to be used as properties. I'll update the docs to indicate that.

Reply:
I'll rename the methods to requestFormAsJson and queryStringAsJson respectively to make it even clearer ... and update the docs.